RR.UnitOfWork/Entities/HRIS/Employee.cs

```diff
@@ -57,6 +58,45 @@ public Employee(EmployeeDto employeeDto, EmployeeTypeDto employeeType)
         EmergencyContactNo = employeeDto.EmergencyContactNo;
         Active = employeeDto.Active;
         InactiveReason = employeeDto.InactiveReason;
+        EmployeeType = new EmployeeType(employeeType);
+        LinkedIn = employeeDto.LinkedIn;
+        CV = employeeDto.CV;
+        PortfolioLink = employeeDto.PortfolioLink;
+        PortfolioPDF = employeeDto.PortfolioPDF;
+        Referral = employeeDto.Referral;
+        HighestQualification = employeeDto.HighestQualification;
+        School = employeeDto.School;
+        QualificationEndDate = employeeDto.QualificationEndDate;
+        BlackListStatus = employeeDto.BlackListStatus;
+        BlackListReason = employeeDto.BlackListReason;
+        IsCandidate = employeeDto.IsCandidate;
+    }
+
```

Comment:
Shouldn't both of these constructors use EmployeeDto? also there's quite some duplication

Reply:
The reason they don't is that the second one is for the Canidate replacement. I basically still make use of the candidate service and dto, but use the Employee model. So I've added the second constructor for use from the candidate service.